src/modules/Telemetry/EnvironmentTelemetry.cpp

```diff
@@ -69,8 +67,7 @@ int32_t EnvironmentTelemetryModule::runOnce()
 {
     if (sleepOnNextExecution == true) {
         sleepOnNextExecution = false;
-        uint32_t nightyNightMs = Default::getConfiguredOrDefaultMs(moduleConfig.telemetry.environment_update_interval,
-                                                                   default_telemetry_broadcast_interval_secs);
+        uint32_t nightyNightMs = Default::getConfiguredOrDefaultMs(moduleConfig.telemetry.environment_update_interval);
```

Comment:
Looks like this line got changed in upstream backmerge accidentally. Everything else looks good to me.